tools/travis/run_tests.sh

```diff
@@ -27,6 +27,10 @@ fi
 
 test="$1"
 
+if [ "$#" -gt 1 ]; then
```

Comment:
Why is this protected by an `if` anyway? Passing it to docker exec if it's empty won't do anything anyway... right?